infinite/index.ts

```diff
@@ -130,14 +128,19 @@ export const infinite = (<Data, Error>(useSWRNext: SWRHook) =>
     // Actual SWR hook to load all pages in one fetcher.
     const swr = useSWRNext<Data[], Error>(
       infiniteKey,
-      async () => {
+      async _infiniteKey => {
```

Comment:
Q: Can we use `infiniteKey` instead of `_infiniteKey`?

Reply:
The type of `infiniteKey` is `string | null`. 

The type and runtime value of `_infiniteKey` here would always be `string`.

We don't need to do extra check later if we use `_infiniteKey` here.

Reply:
can we rename it? `_xx` feels like a arg that usually being skipped in ts. like `stringifiedKey`